### PR TITLE
grpc 1.52.1

### PR DESCRIPTION
**Pull request by ZhongRuoyu**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by ZhongRuoyu** [comment]
Note: `mavsdk`'s tag `v1.4.12` was revoked.

File: Formula/mavsdk.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Mavsdk < Formula
   desc "API and library for MAVLink compatible systems written in C++17"
   homepage "https://mavsdk.mavlink.io"
   url "https://github.com/mavlink/MAVSDK.git",
-      tag:      "v1.4.12",
-      revision: "1e08fccd593b25970b87df9f2609ce4907ceae1f"
```
Comment (chenrui333) [inline Formula/mavsdk.rb:8]: this wont work. it needs `version_scheme 1`

Comment (chenrui333) [reply Formula/mavsdk.rb:8]: Can we also have some upstream issue to understand why the tag got removed?

**Comment by chenrui333** [comment]
- https://github.com/mavlink/MAVSDK/issues/1978

File: Formula/mavsdk.rb (hunk 1 above)
Comment (ZhongRuoyu) [reply Formula/mavsdk.rb:8]: OK, I opened https://github.com/mavlink/MAVSDK/issues/1978. Should `version_scheme 1` be added now, or should we wait for upstream's response?

Comment (chenrui333) [reply Formula/mavsdk.rb:8]: let's wait a bit for the upstream feedback on the tag thing first :)

Comment (chenrui333) [reply Formula/mavsdk.rb:8]: anyway, I saw you already updated it, should be good now. (that also works!)

**Comment by chenrui333** [comment]
let's wait a bit for the `mavsdk`'s response before the merge.

**Comment by chenrui333** [comment]
upstream would make a new release tomorrow for `mavsdk`, I think that would be a better fix for this PR bump.

**Comment by ZhongRuoyu** [comment]
`mavsdk` has retagged `v1.4.12` (https://github.com/mavlink/MAVSDK/issues/1978#issuecomment-1425020366), so I bumped the revision number of `mavsdk` instead.

**Comment by chenrui333** [comment]
Thanks @ZhongRuoyu!!

**Comment by chenrui333** [comment]
[the previous run](https://github.com/Homebrew/homebrew-core/actions/runs/4139807063/jobs/7164257062) had some issue, rebased to the latest master and do another run 🤞

**Comment by ZhongRuoyu** [comment]
https://github.com/Homebrew/homebrew-core/actions/runs/4143679825/jobs/7165970326#step:6:1

The build didn't happen at all :(

```plaintext
mkdir: bottles: File exists
Error: Process completed with exit code 1.
```

**Comment by chenrui333** [comment]
The build was OK last week, https://github.com/Homebrew/homebrew-core/actions/runs/4091077688/jobs/7054971310

**Comment by ZhongRuoyu** [comment]
Extracted from 12-arm64 logs:

```plaintext
CMake Error at cmake_modules/FindProtobuf.cmake:88 (message):
  Could not find Protobuf.  Looked in /opt/homebrew.
Call Stack (most recent call first):
  cmake_modules/ThirdpartyToolchain.cmake:376 (find_package)
  CMakeLists.txt:138 (INCLUDE)
```

An error also seen in #121566.

**Comment by carlocab** [comment]
Cherry-picked onto #123507.

**Pull request closed without merging**